api/src/pcapi/notifications/push/transactional_notifications.py

```diff
@@ -81,3 +83,17 @@ def get_offer_notification_data(user_id: int, offer: Offer) -> TransactionalNoti
         ),
         extra={"deeplink": offer_webapp_link(offer)},
     )
+
+
+def get_unretrieved_bookings_with_offers_notification_data(booking: Booking) -> TransactionalNotificationData:
+    remaining_time = datetime.now() - booking.expirationDate
```

Comment:
```suggestion
    remaining_time = datetime.utcnow() - booking.expirationDate
```

Ça ne change rien en prod (où la timezone est UTC), mais en local c'est probablement différent. (Ou peut-être pas puisque le nombre de jours est probablement le même, mais bon, mieux vaut prendre les bonnes habitudes et utiliser `utcnow()` partout.)